stock_landed_cost_backdating/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 Ecosoft Co., Ltd (https://ecosoft.co.th)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Stock Landed Cost Backdating",
+    "summary": """
+        This module allows to fill Landed Cost date to backdate (date != now),
+    Stock valuation date will following to accouting date in Journal Entry on Landed Cost.""",
+    "version": "15.0.1.0.0",
+    "license": "AGPL-3",
```

Comment:
It seems your headers are declared as LGPL, so pick one and update either headers or this.